Formula/sftpgo.rb

```diff
@@ -0,0 +1,70 @@
+class Sftpgo < Formula
+  desc "Fully featured SFTP server with optional HTTP/S, FTP/S and WebDAV support"
+  homepage "https://github.com/drakkan/sftpgo"
+  url "https://github.com/drakkan/sftpgo/releases/download/v2.2.3/sftpgo_v2.2.3_src_with_deps.tar.xz"
+  sha256 "6c8676725e86ee3f6ad46a340a84f0da37cab8b6ea7b6aee86b2b96ba5e6671a"
+  license "AGPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-trimpath", *std_go_args(ldflags: "-s -w"), "-o", "sftpgo"
```

Comment:
`-trimpath` — is the one of  std_go_args, `-o` by default points to `#{bin}/sftpgo`

```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w")
```